Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -422,6 +422,11 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
         return obj;
     }
 
+    public bool WorkspotIsInternal()
```

Comment:
The primary problem is that it also needs to be aware of the size of the furniture (or the furniture itself to get the size from) and it isn't currently, and otherwise doesn't need to be aware of.